src/sections/faq.liquid

```diff
@@ -0,0 +1,154 @@
+<section class="faq{% if section.settings.background_color_preset != 'none' and section.settings.background_color_preset != blank %} {{ section.settings.background_color_preset | prepend: 'section-color-preset__' }}{% endif %}" data-preview-item="faq">
+  <div class="faq__container">
+    {% if section.settings.heading != blank %}
+      <header class="section__headings">
+        <h3 class="section__heading">{{ section.settings.heading }}</h3>
+        {% if section.settings.subheading != blank %}
```

Comment:
Is it possible to set the subheading when there is no heading?  Would this be confusing if `section.settings.heading` is blank but subheading isn't and they don't see either?

Reply:
If we had the ability to do conditional fields, that would be amazing. It would resolve the workarounds that we have used in the past, to really help clear up any UI confusion we run into now. However, in the case of the subheadings, they should always follow a heading. We will have to add more to the schema description to clear this up. When the formal QA starts, I'll ask Marika to help address this :)